app/code/Magento/Sales/Api/Data/OrderPaymentInterface.php

```diff
@@ -261,7 +261,7 @@ public function getAdditionalData();
     /**
      * Gets the additional information for the order payment.
      *
-     * @return string[] Array of additional information.
+     * @return array|null|mixed Array of additional information.
```

Comment:
Thank you for the deep explanation. I was expecting this kind of response. Let's wait for the new version then.